src/detections/chart/loading.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import CircularProgress from '@material-ui/core/CircularProgress'
```

Comment:
Let's combine these 2 imports, since they come from the same package.

Reply:
done